ADA/git_databricks.qmd

```diff
@@ -52,29 +46,26 @@ The connection between git and Databricks is established using a secure access t
 
```

Comment:
Missed a git to Git capitalisation on line 45, there's two on there

Reply:
Oh my local version has this correctly, I wonder if I didn't commit it somehow? 
<img width="379" alt="image" src="https://github.com/user-attachments/assets/c9514d0a-c6d6-40d9-b37f-26dec63d0389">